apps/codeprop/Makefile.codeprop-tmp

```diff
@@ -0,0 +1,9 @@
+codeprop-tmp_src = codeprop-tmp.c
+
+# Enable large memory model supports for WISMOTE and EXP5438 platform
```

Comment:
I can also test for the Z1 mote, but I don't like platform specific defines... maybe we could check if CPU_HAS_MSP430X (the default case) is defined, then override with CPU_HAS_MSP430X_LARGE? or check if CPU_HAS_MSP430X_LARGE is not defined and throw an error? what do you think it would be the best approach @simonduq ?